provider/aws/formation/rack.json

```diff
@@ -502,6 +503,11 @@
       "MinValue": "1",
       "Type": "Number"
     },
+    "InstancePolicy": {
+      "Default": "",
+      "Description": "ARN of an additional IAM policy to add to the cluster instances",
+      "Type": "String"
```

Comment:
Does it make any sense to make this a list so you could attach multiple (more granular) policies?

Reply:
I can't think of an easy way to merge a user-specified list with a static list inside the template.